libs/react-helpers/src/lib/grid/price-cell.tsx

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
 import { getDecimalSeparator } from '../format';
-export interface IPriceCellProps {
+export interface AgPriceCellProps {
   value: number | bigint | null | undefined;
   valueFormatted: string;
   testId?: string;
 }
 
-export const PriceCell = React.memo(
-  ({ value, valueFormatted, testId }: IPriceCellProps) => {
+export const AgPriceCell = React.memo(
```

Comment:
chore (blocking): This component is not always used within ag grid so renaming here doesn't make sense